assets/scss/components/_features.scss

```diff
@@ -0,0 +1,55 @@
+.section--features {
+	padding-top:45px; 
+	padding-bottom:145px; 
+	.feature-title {
+		text-align:center;
+	}
+	.feature {
```

Comment:
`.feature` in this case looks like its own component, which means these styles should be within a feature.scss file